src/ZString.Unity/Assets/Scripts/ZString/FormatHelper.cs

```diff
@@ -131,15 +131,15 @@ private static void FormatToRightJustify<TBufferWriter, T>(ref TBufferWriter sb,
             if (typeof(T) == typeof(string))
             {
                 var s = Unsafe.As<string>(arg);
-                int padding = width - s.Length;
+                int padding = width - s!.Length;
                 if (padding > 0)
                 {
                     var paddingSpan = sb.GetSpan(padding);
                     paddingSpan.Fill(sp);
                     sb.Advance(padding);
                 }
 
-                ZString.AppendChars(ref sb, s.AsSpan());
+                ZString.AppendChars(ref sb, s!.AsSpan());
```

Comment:
Redundant when the initialization of `s` is adjusted - `var s = Unsafe.As<string>(arg)!;`